src/Main/Environment.hs

```diff
@@ -126,20 +127,28 @@ testProcess check defaultMsg testName prog args inp maudeTest = do
 ensureGraphVizDot :: Arguments -> IO Bool
 ensureGraphVizDot as = do
     putStrLn $ "GraphViz tool: '" ++ dot ++ "'"
-    testProcess check errMsg " checking version: " dot ["-V"] "" False
+    testProcess check errMsg1 " checking version: " dot ["-v </dev/null"] "" False
   where
     dot = dotPath as
     check _ err
-      | "graphviz" `isInfixOf` map toLower err = Right $ init err ++ ". OK."
-      | otherwise                              = Left  $ errMsg
-    errMsg = unlines
+      | "graphviz" `isInfixOf` err' && "png" `isInfixOf` err' = Right $ init err ++ ". OK."
```

Comment:
Should we tighten this check, and make sure that `png` occurs on the `device` line, surrounded by whitespace?